### ISSUE TITLE
False positive missing dependency with SCSS files on Windows

### ISSUE THREAD

**Issue by ProchaLu** [depcheck/depcheck#838]
### Bug Description

Depcheck falsely reports a missing dependency for SCSS files used for styling in a create-react-app project on Windows machines. This issue occurs during our [UpLeveled/Preflight](https://github.com/upleveled/preflight) check, which uses Depcheck. The following error message is displayed:

```
Missing dependencies found:
* C: .\src\ App.module.scss
```

To reproduce this issue with Windows and Depcheck, I created a small [reproduction](https://github.com/ProchaLu/depcheck-repro).

For people without a Windows machine, I added a GitHub workflow that installs all dependencies and runs the depcheck command. The workflow fails with the same error in the [GitHub Actions run](https://github.com/ProchaLu/depcheck-repro/actions/runs/6403483873/job/17382391369):

```
[FAILED] Missing dependencies found:
        * D: .\src\App.module.scss
```

Where the dependency is listed in package.json:

```
  "dependencies": {
    "@upleveled/react-scripts": "5.0.12",
    "jest-environment-jsdom": "^29.7.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "sass": "^1.68.0"
  },
    "devDependencies": {
    "@jest/globals": "^29.7.0",
    "@testing-library/jest-dom": "^5.17.0",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "@types/react": "^18.2.24",
    "@types/react-dom": "^18.2.8",
    "@types/testing-library__jest-dom": "^5.14.9",
    "eslint": "^8.50.0",
    "eslint-config-upleveled": "^5.0.5",
    "stylelint": "^15.10.3",
    "stylelint-config-upleveled": "^1.0.2",
    "typescript": "^5.2.2"
  },
```

### Versions

- `node -v`: v18.17.1
- `pnpm -v`: 8.3.1
- `depcheck --version`: 1.4.6

### `depcheck --json`:
```
{"dependencies":["depcheck","jest-environment-jsdom","sass"],"devDependencies":["@testing-library/user-event","@types/testing-library__jest-dom","stylelint","stylelint-config-upleveled"],"missing":{"@typescript-eslint/parser":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"@next/eslint-plugin-next":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"@typescript-eslint/eslint-plugin":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-import":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-jsx-a11y":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-jsx-expressions":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-react":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-react-hooks":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-security":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-sonarjs":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-unicorn":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-upleveled":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-import-resolver-typescript":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"]},"using":{"eslint-config-upleveled":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs","/Users/lukas/Documents/projects/depcheck-repro/tsconfig.json"],"@typescript-eslint/parser":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"@next/eslint-plugin-next":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"@typescript-eslint/eslint-plugin":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-import":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-jsx-a11y":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-jsx-expressions":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-react":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-react-hooks":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-security":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-sonarjs":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-unicorn":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-plugin-upleveled":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"eslint-import-resolver-typescript":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs"],"@types/react":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs","/Users/lukas/Documents/projects/depcheck-repro/tsconfig.json"],"@types/react-dom":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs","/Users/lukas/Documents/projects/depcheck-repro/tsconfig.json"],"eslint":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs","/Users/lukas/Documents/projects/depcheck-repro/tsconfig.json"],"typescript":["/Users/lukas/Documents/projects/depcheck-repro/.eslintrc.cjs","/Users/lukas/Documents/projects/depcheck-repro/package.json","/Users/lukas/Documents/projects/depcheck-repro/tsconfig.json"],"@upleveled/react-scripts":["/Users/lukas/Documents/projects/depcheck-repro/package.json"],"react":["/Users/lukas/Documents/projects/depcheck-repro/package.json","/Users/lukas/Documents/projects/depcheck-repro/package.json","/Users/lukas/Documents/projects/depcheck-repro/src/App.test.js","/Users/lukas/Documents/projects/depcheck-repro/src/index.js"],"@jest/globals":["/Users/lukas/Documents/projects/depcheck-repro/src/App.test.js"],"@testing-library/react":["/Users/lukas/Documents/projects/depcheck-repro/src/App.test.js"],"react-dom":["/Users/lukas/Documents/projects/depcheck-repro/src/App.test.js","/Users/lukas/Documents/projects/depcheck-repro/src/index.js"],"@testing-library/jest-dom":["/Users/lukas/Documents/projects/depcheck-repro/src/setupTests.js"]},"invalidFiles":{},"invalidDirs":{}}
```

**Comment by ProchaLu** [comment]
As a temporary workaround, it might be possible to exclude specific files and add an `--ignores-patterns=` argument to ignore the file pattern `*.module.scss`

**Comment by ProchaLu** [comment]
Fixed in Depcheck `v1.4.6`
https://github.com/depcheck/depcheck/releases/tag/v1.4.6
